app/javascript/components/request-workflow-status/data.js

```diff
@@ -0,0 +1,65 @@
+/** Types of workflow state status */
+export const workflowStateTypes = {
+  success: { text: 'success', tagType: 'green' },
+  error: { text: 'error', tagType: 'red' },
+  failed: { text: 'failed', tagType: 'gray' },
+  pending: { text: 'pending', tagType: 'gray' },
+};
+
+/** Function to get the header data of workflow states table. */
+const headerData = () => ([
+  {
+    key: 'name',
+    header: __('Name'),
+  },
+  {
+    key: 'enteredTime',
+    header: __('Entered Time'),
+  },
+  {
+    key: 'finishedTime',
+    header: __('Finished Time'),
+  },
+  {
+    key: 'duration',
+    header: __('Duration'),
+  },
+]);
+
+// const convertDate = (date) => `${moment(date).format('MM/DD/YYYY')} ${moment(date).format('h:mm:ss A')}`;
+const convertDate = (date) => {
+  const utcDate = new Date(date);
```

Comment:
I assume this is the moment stuff you mentioned?  In a follow up we try to move to using moment?

Reply:
yes